src/main/resources/mybatis/collections.xml

```diff
@@ -102,4 +102,48 @@
         ) as collection
     </select>
 
+
+    <select id="getStatisticalTimeSeriesJson" parameterType="String" resultType="String">
+        select row_to_json(fsts)
+          from ( select 'Feature' as "type",
+                          ml.monitoring_location_identifier as "id",
+                          json_build_object(
+                             'type', 'Point',
+                             'coordinates', json_build_array(
+                                 to_json(trunc(st_x(st_transform(geom, 4326))::numeric, 7)),
+                                 to_json(trunc(st_y(st_transform(geom, 4326))::numeric, 7))
+                             )
+                          ) as "geometry",
+                          json_build_object('samplingFeatureName', site_name) as "properties",
+                          ( select array_to_json(array_agg(sts))
+                              from ( select distinct 
+                                            groundwater_daily_value_identifier as "id",
+                                            observered_property_name||', '||statistic as "description",
+                                            observered_property_id as "parameterCode",
+                                            statistic_id as "statisticCode",
+                                            #{serverBaseURL,jdbcType=VARCHAR}
+                                                ||'collections/monitoring-locations/items/'
+                                                ||monitoring_location_identifier
+                                                ||'/observations/statistical-time-series/'
+                                                ||time_series_unique_id
+                                                   as "url"
+                                       from nwis.groundwater_daily_value gw
+                                      where gw.monitoring_location_id = ml.monitoring_location_id
+                                   ) as sts
+                          ) as "timeSeries"
+                   from nwis.monitoring_location ml
+                  where geom is not null
+                    and ml.monitoring_location_identifier = #{featureId,jdbcType=VARCHAR}
+                    and ('monitoring-locations' = #{collectionId,jdbcType=VARCHAR}
```

Comment:
I still don't like ignoring the capability of mybatis to handle this condition and simplify the actual query being submitted.

Reply:
That would create two queries in the processing path of the pg database. I have been told that it is best to have as few processing paths (Oracle calls them explain plans, and yes I know you know what those are is why I am calling them out to you) as possible. The mybatis if clauses will create a new query. In this case it would be only two but I thought we could optimize later if it was necessary. I would also like to make this clause a named one so that we could use it in multiple places. Can we name one in on file and use it in other files. Right now I have the same where clause in two different files.

Reply:
discussion tabled to IOW-431